curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-primes.english.md

```diff
@@ -8,9 +8,14 @@ forumTopicId: 16085
 
 ## Description
 <section id='description'>
-Sum all the prime numbers up to and including the provided number.
-A prime number is defined as a number greater than one and having only two divisors, one and itself. For example, 2 is a prime number because it's only divisible by one and two.
-The provided number may not be a prime.
+
+A <dfn>prime number</dfn> is a whole number greater than 1 with exactly two divisors: 1 and
+itself. For example, 2 is a prime number because it is only divisible by 1 and 2. In
+contrast, 4 is not prime since it is divisible by 1, 2 and 4.
+
+Rewrite `sumPrimes` so it returns the sum of all prime numbers that are less than or
```

Comment:
Why don't we move this to the `Instructions` section?

Reply:
There's an explanation on the suggestion of yours I resolved and an agreement in the comment above this.

Reply:
Well, as I said, it's better, but it makes more sense to me to fix those all in one PR.  We can do it piecemeal if you'd rather, though.

Reply:
How about we fix it for this one and create an issue with the `help wanted` label and someone else can move them?  If no one creates a PR by next week, I will do it.

Reply:
Sure, that's fine by me.

Reply:
I would rather merge this as-is. If you are planning on making the bulk changes, it won't be tough to do it to this one as well. And if, for some reason, you don't get around to it for a while, they will remain consistent.

Reply:
@moT01 OK.  go ahead and merge this one.